recipe/migrations/libspatialindex-1.9.2.yaml

```diff
@@ -0,0 +1,17 @@
+# To learn more about migrations read CFEP-09
+# https://github.com/conda-forge/conda-forge-enhancement-proposals/blob/master/cfep-09.md
+# The timestamp of when the migration was made
+# Can be obtained by copying the output of
+# python -c "import time; print(f'{time.time():.0f}')"
+migrator_ts: 1572370591
+__migrator:
+  kind:
+    version
+  # Only change the migration_number if the bot messes up,
+  # changing this will cause a complete rerun of the migration
+  migration_number:
+    1
+
+# The name of the feedstock you wish to migrate
+libspatialindex:
+  - 1.9.3
```

Comment:
@CJ-Wright is this correct? Is this enough? If so I'll write some docs about it.

Reply:
Yes, although you can also add the `pin_run_as_build` section here as well. Also you don't need to update the config/recipe, since that will push out the new pinnings before we've built them out.

Reply:
see https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/273/files#diff-75576eba2a574dd4bce625fa345716b9L5 for an example with pin run as build

Reply:
> Yes, although you can also add the `pin_run_as_build` section here as well.

Thanks for the example.

> Also you don't need to update the config/recipe, since that will push out the new pinnings before we've built them out.

I was wondering about that. Maybe we should split the config and the migration repository to avoid people editing them together? Probably not worth it?

Reply:
> Also you don't need to update the config/recipe, since that will push out the new pinnings before we've built them out.

It's already pushed out since it is a looser pin.